openshift-hack/images/os/install.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -xeou pipefail
+
+yum install -y ostree rpm-ostree yum-utils selinux-policy-targeted
+curl http://base-4-10-rhel8.ocp.svc > /etc/yum.repos.d/rhel8.repo
+curl http://base-4-10-rhel8-ocp.svc/rhel-8-advanced-virt/ > /etc/yum.repos.d/rhel8-advanced-virt.repo
+curl http://base-4-10-rhel8-ocp.svc/rhel-8-fast-datapath-aarch64/ > /etc/yum.repos.d/rhel8-fast-datapath-aarch64.repo
+curl http://base-4-10-rhel8-aarch64.ocp.svc > /etc/yum.repos.d/rhel8-aarch64.repo
```

Comment:
You've got a mix of `base-4-10-rhel8.ocp.svc` and `base-4-10-rhel8-ocp.svc` as hostnames

I think you want all `base-4-10-rhel8.ocp.svc`